support-models/src/main/scala/com/gu/support/promotions/Promotion.scala

```diff
@@ -61,6 +61,6 @@ object Promotions {
     starts = new DateTime(1971, 2, 20, 12, 0, 0, 0),
     expires = None,
     discount = None, freeTrial = None, incentive = None,
-    introductoryPrice = Some(IntroductoryPriceBenefit(price = 6, periodLength = 7, periodType = Issue))
+    introductoryPrice = Some(IntroductoryPriceBenefit(6, 6, Issue))
```

Comment:
I would recommend to keep the field names still for clarity, as two 6s next to each other is hard to understand
```suggestion
    introductoryPrice = Some(IntroductoryPriceBenefit(price = 6, periodLength = 6, periodType = Issue))
```